src/Resources/views/Error/error_layout.html.twig

```diff
@@ -0,0 +1,83 @@
+{% spaceless %}
```

Comment:
Well, the `AcmeDemoBundle` notes `{% extends 'AcmeDemoBundle::layout.html.twig' %}` and as we are not the "main" framework template but only the frame for errors, I prefixed it with `error_`